diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index a74e79afd0e9e885d05c6fd623416347378d6b84..0000000000000000000000000000000000000000
diff --git a/dist/module.mjs b/dist/module.mjs
index af5d831e1e8113e9b2739460315ea3f07c3c707e..23e76b845f002e25f0a00e43c8aeb43953fa36ad 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -22,13 +22,14 @@ const plugin = (options, nuxtOptions) => ({
   name: "vite-plugin-nuxt-monaco-editor",
   enforce: "pre",
   resolveId(src) {
-    if (src.includes("monaco-editor/esm/vs/") && src.endsWith(".js?worker")) {
+    if (/monaco-editor\/esm\/vs\/.*\.worker\.js/.test(src)) {
       return resolveModule(
         src.replace("?worker", "").replace("__skip_vite", "").replace("node_modules", "").replace(nuxtOptions.app.baseURL, "/").replace(/\/\/+/g, "/").replace(/^\//, "")
       );
     }
   },
   async load(id) {
+    if (id[0] === '\0') return
     id = id.split("?")[0];
     const vsPath = id.includes("monaco-editor/esm") ? id.split("monaco-editor/esm/").pop() : null;
     if (options.locale !== "en") {
